iree/test/e2e/linalg_ext_ops/BUILD

```diff
@@ -46,13 +49,33 @@ iree_check_single_backend_test_suite(
             "scan.mlir",
         ],
         include = ["*.mlir"],
+        # linalg_transform_spec is a an MLIR file that specifies a
+        # transformation, it needs to be included as data.
```

Comment:
can drop this comment, or rephrase to mention why it's excluded - it's using lit and not check

moving to a new directory (linalg_transform? there aren't any linalg_ext ops in the test) would also get around the glob exclude awkwardness (right now these files appear alongside test failures, when this exclusion is actually intentional)

Reply:
This was in the wrong place. Putting in at the right place (below where you suggested in the other comment)